core/src/actors/epoch_manager/mod.rs

```diff
@@ -28,17 +36,32 @@ pub enum EpochManagerError {
     Overflow,
 }
 
-/// EpochManager actor
-#[derive(Debug, Default)]
+////////////////////////////////////////////////////////////////////////////////////////
+// ACTOR BASIC STRUCTURE
+////////////////////////////////////////////////////////////////////////////////////////
+/// Epoch manager actor
+#[derive(Default)]
 pub struct EpochManager {
+    /// Timestamp of checkpoint #0 (the second in which epoch #0 started)
     checkpoint_zero_timestamp: Option<i64>,
+
+    /// Period between checkpoints, in seconds
     checkpoints_period: Option<u16>,
+
+    /// Subscriptions to a particular epoch
+    subscriptions_epoch: BTreeMap<Epoch, Vec<Box<dyn SendableNotification>>>,
+
+    /// Subscriptions to all epochs
+    subscriptions_all: Vec<Box<dyn SendableNotification>>,
+
+    /// Last epoch that was checked by the epoch monitor process
+    last_checked_epoch: Option<Epoch>,
 }
 
-/// Required trait for being able to retrieve EpochManager address from system registry
+/// Required trait for being able to retrieve `EpochManager` address from system registry
 impl actix::Supervised for EpochManager {}
 
-/// Required trait for being able to retrieve EpochManager address from system registry
+/// Required trait for being able to retrieve `EpochManager` address from system registry
```

Comment:
I think from file to file we should try to be consistent with the use of nouns and the use of ``. It's fine either way we use them, as long as we're fully consistent within individual files.